### PR TITLE
Tribute Page Linters

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
Fix webhint errors
Fix eslint errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,53 +2,26 @@
...
-
-      }
-    </style>
   </head>
 
   <body>
-    <main id= "main">
+    <main id="main">
       <h1 id="title" class="nel">Dr. Nelson Mandela</h1>
       <p class="des">A man with honor</p>
```
Comment (sharon-odhiambo) [inline index.html:13]: Hi, kindly update your page to stick to the template. 
I'd suggest using the center tag in your html
<center><h1 id="title" class="nel">Dr. Nelson Mandela</h1>
      <p class="des">A man with honor</p></center>

File: index.html [hunk 2]
```diff
@@ -2,53 +2,26 @@
...
-    <figure>
-      <id="img-div">
-      <img id="image" class="center" image src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/02/Nelson_Mandela_1994.jpg/220px-Nelson_Mandela_1994.jpg" 
-      alt="img-caption">Nelson Mandela</id>
-      <figcaption>Nelson Mandela in his office during his presidency</figcaption>
+    <figure class="img-div">
+      <img class="center" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/02/Nelson_Mandela_1994.jpg/220px-Nelson_Mandela_1994.jpg" 
+      alt="Nelson Mandela in his day to day activities"/>
+      <figcaption class="img-caption">Nelson Mandela in his office during his presidency</figcaption>
     </figure>
```
Comment (sharon-odhiambo) [inline index.html:18]: Please use a different background color for this section as used in the template

File: index.html [hunk 3]
```diff
@@ -2,53 +2,26 @@
...
-    <section id= "tribute info">
-      <h3 id="headline">This is Nelson Mandela's tribute</h3>
+    <section class="tribute info">
+      <h3 class="headline">This is Nelson Mandela's tribute</h3>
       <h4>
-        <p>Nelson Rolihlahla Mandela (18 July 1918 – 5 December 2013) was a South African anti-apartheid revolutionary political leader who served as the first president of South Africa from 1994 to 1999</p>
+      Nelson Rolihlahla Mandela (18 July 1918 – 5 December 2013) was a South African anti-apartheid revolutionary political leader who served as the first president of South Africa from 1994 to 1999
       </h4>
```
Comment (sharon-odhiambo) [inline index.html:23]: Please center this section well.
You could also add a few more info using ul tag
Kindly change the font-weight for p

File: index.html [hunk 4]
```diff
@@ -2,53 +2,26 @@
...
-      <a id="tribute link" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target= "_blank">Nelson tribute link</a>
+      <a class="tribute link" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target="_blank" rel="noopener">Nelson tribute link</a>
```
Comment (sharon-odhiambo) [inline index.html:24]: Kindly check your code. You have an equal sign at the end that's showing up below this link.
I'd also suggest changing the font-type and weight to match the template.

**Review by sharon-odhiambo — COMMENTED** [review]
# Changes Required 
Hi @sharon-odhiambo 
You're almost there!

**Highlights**
Good job with the linters.
Great commit messages.

**Required Changes** 
Kindly check the inline comments to make the required changes

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @sharon-odhiambo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [ ] KIndly add a reason why the change should be implemented, this always helps them understand what a change is required.
image

- [ ] Kindly make your comments as clear as possible, for example, you could phrase your comment like this, "Its a requirement to follow the given design, kindly do so by using the given fonts, colors, and spacing"
image


image
- [ ] Again in the image above, I would advise you to make your comment clear and also add a reason why the change is required. I would say something like, "I would advise you to use a `ul` tag to group your items, with each item wrapped in a `li` tag, this is the correct semantic tag"

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sharon-odhiambo) [inline index.html:13]: Hello, kindly ensure you use the template's design, spacing and color as indicated in the instructions.
For instance, I'd suggest you center this section to achieve the required result.

File: index.html (hunk 2 above)
Comment (sharon-odhiambo) [inline index.html:18]: While you've done a good job in this section, the instructions require you to stick to the template. I'd suggest changing your background color for this section to obtain the required result.

File: index.html (hunk 3 above)
Comment (sharon-odhiambo) [inline index.html:23]: Good job with this section again. However, I'd suggest checking the required design to ensure they look the same.
In this part, you could center your it and use the **ul** tag to group the items and the **li** tag to wrap each item. Additionally, I'd suggest adding more information with **li** tag wraps.

**Review by sharon-odhiambo — COMMENTED** [review]
Hello@sharon-odhiambo

Good job so far. 
You're almost there.

**Highlights** 
Descriptive README
Descriptive commit messages
Good job with the linters-no linter errors.

Required Changes 
While you've a very good job, I'd suggest some changes before you proceed to the next project.

Kindly check the inline comments.

**Review by m05e5 — APPROVED** [review]
# Hi @sharon-odhiambo , 

### Status: Approved ✔️
goodjob


:+1: . Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**